api/types/api.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
```

Comment:
We should have a distinction between state APIs and observability APIs. Whether that be g naming convention, separate files, or both.